src/standalone/Vegetation/component_models.jl

```diff
@@ -123,11 +123,50 @@ may be needed and passed in (via the `canopy` model itself).
 The right hand side for the entire canopy model can make use of
 these functions for the individual components.
 """
-function ClimaLand.make_compute_exp_tendency(::AbstractCanopyComponent, canopy)
-    function compute_exp_tendency!(dY, Y, p, t) end
+function ClimaLand.make_compute_exp_tendency(
+    component::AbstractCanopyComponent,
+    canopy,
+)
+    function compute_exp_tendency!(dY, Y, p, t)
+        vars = prognostic_vars(component)
+        dY_canopy = getproperty(dY, name(canopy))
+        if !isempty(vars)
+            getproperty(dY_canopy, name(component)) .= 0
+        end
+    end
     return compute_exp_tendency!
 end
 
+"""
+     ClimaLand.make_compute_imp_tendency(component::AbstractCanopyComponent, canopy)
+
+Creates the compute_imp_tendency!(dY,Y,p,t) function for the canopy `component`.
+
+Since component models are not standalone models, other information
+may be needed and passed in (via the `canopy` model itself).
+The right hand side for the entire canopy model can make use of
+these functions for the individual components.
+"""
+function ClimaLand.make_compute_imp_tendency(
+    component::AbstractCanopyComponent,
+    canopy,
+)
+    function compute_imp_tendency!(dY, Y, p, t)
+        vars = prognostic_vars(component)
+        dY_canopy = getproperty(dY, name(canopy))
+        if !isempty(vars)
+            getproperty(dY_canopy, name(component)) .= 0
+        end
+
+    end
+    return compute_imp_tendency!
+end
+
+
+function make_compute_jacobian(component::AbstractCanopyComponent, canopy)
+    function compute_jacobian!(W, Y, p, dtγ, t) end
```

Comment:
For the implicit and explicit tendencies, we set dY = 0 by default (above). Here, do we need to set elements of the jacobian to zero? I think not, but Im just noticing a difference in how we handle it.

Reply:
and so want to double check

Reply:
I also don't think we need to zero them out, but not 100% sure. @dennisYatunin do you have any insights?

Reply:
We talked about this offline and clarified that the blocks in the jacobian for explicitly-stepped variables will always be `-I`. There's nowhere else these blocks would be updated, so we expect them to remain unchanged and don't need to reset any blocks of the jacobian here